server/order/order.go

```diff
@@ -75,8 +130,19 @@ type Order interface {
 	// Time returns the Order's server time, when it was received by the server.
 	Time() int64
 
+	// FilledAmt returns the filled amount of the order.
+	FilledAmt() uint64
+
 	// Remaining computes the unfilled amount of the order.
 	Remaining() uint64
+
+	// Base returns the unique integer identifier of the base asset as defined
+	// in the asset package.
+	Base() uint32
+
+	// Quote returns the unique integer identifier of the quote asset as defined
+	// in the asset package.
+	Quote() uint32
```

Comment:
This is getting out of hand I think.  Will look into avoiding a getter for every piece of info in an order.

Reply:
Hmm, there could be a helper function like `OrderMarket(ord order.Order) (base, quote uint32, error)` to pull the quote and base asset index via a type switch, but I think this is fine for now.